packages/dds/tree/src/feature-libraries/modular-schema/schemaChangeFamily.ts

```diff
@@ -0,0 +1,278 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { assert } from "@fluidframework/common-utils";
+import { ChangeEncoder, ChangeFamily, JsonCompatibleReadOnly, ProgressiveEditBuilder } from "../../change-family";
+import { ChangeRebaser } from "../../rebase";
+import {
+    FieldSchema,
+    GlobalFieldKey,
+    LocalFieldKey,
+    SchemaData,
+    TreeSchema,
+    TreeSchemaIdentifier,
+} from "../../schema-stored";
+import { AnchorSet, Delta, FieldKey } from "../../tree";
+import { brand, Invariant, Multimap } from "../../util";
+import { FieldChangeHandler, FieldChangeMap, FieldChangeset, NodeChangeset } from "./fieldChangeHandler";
+import { FullSchemaPolicy } from "./fieldKind";
+
+export class SchemaChangeFamily implements
```

Comment:
"SchemaChangeFamily" sounds as though we're changing the schema. How about `NodeChangeFamily`?

Reply:
While I'm not attached to the name, the idea is that this is a ChangeFamily which delegates to the field kinds as described in the schema. I don't think `NodeChangeFamily` is very descriptive. What do you think of the name `ModularChangeFamily`?

Reply:
`NodularChangeFamily`? Just kidding, `ModularChangeFamily` sounds good.